packages/lesswrong/components/notifications/NotificationsPage/NotificationsPageItem.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { FC, ReactNode } from "react";
+import { Components, registerComponent } from "../../../lib/vulcan-lib";
+import { AnalyticsContext } from "../../../lib/analyticsEvents";
+import { useSingle } from "../../../lib/crud/withSingle";
+import type { ForumIconName } from "../../common/ForumIcon";
+import classNames from "classnames";
+
+const ICON_WIDTH = 24;
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 14,
+    fontWeight: 500,
+    color: theme.palette.grey[600],
+  },
+  container: {
+    display: "flex",
+    gap: "8px",
+  },
+  iconContainer: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    color: theme.palette.text.alwaysWhite,
+    borderRadius: "50%",
+    minWidth: ICON_WIDTH,
+    width: ICON_WIDTH,
+    height: ICON_WIDTH,
+    "& svg": {
+      width: 14,
+      height: 14,
+    },
+  },
+  iconPrimary: {
+    backgroundColor: theme.palette.primary.main,
+  },
+  iconGrey: {
+    backgroundColor: theme.palette.icon.recentDiscussionGrey,
+  },
+  iconYellow: {
+    color: theme.palette.icon.headerKarma,
+    backgroundColor: "transparent",
+    transform: "scale(1.5)",
+  },
+  iconClear: {
+    color: theme.palette.primary.main,
+    backgroundColor: "transparent",
+    transform: "scale(1.5)",
+  },
+  meta: {
+    marginBottom: 12,
+    lineHeight: "1.5em",
+    fontWeight: 500,
+    maxWidth: "100%",
+  },
+  hideOnMobile: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+  },
+  preview: {
+    flexGrow: 1,
+    minWidth: 0,
+  },
+});
+
+export type IconVariant = "primary" | "grey" | "yellow" | "clear";
+
+export const NotificationsPageItem = ({
```

Comment:
This is now just a styling wrapper that's used internally by `NotificationsPageKarmaChange` and `NotificationsPageNotification`.